src/slices/columnSlice.ts

```diff
@@ -101,3 +102,5 @@ export const {
   removeCard,
   moveCard,
 } = columnsSlice.actions;
+
+export const selectColumns = (state: RootState) => state.columns.columns;
```

Comment:
you have selector with same issue as it was in component. Your redux state should not have columns.columns, only state.columns